src/itty-router.ts

```diff
@@ -59,11 +59,12 @@ const toQuery = (params: any) =>
             : acc[k] = [...[acc[k]], v].flat()
     ) && acc || acc, {})
 
-export const Router = ({ base = '', routes = [] }: RouterOptions = {}): RouterType =>
+export const Router = <RequestType = RequestLike>({ base = '', routes = [] }:
```

Comment:
I think this default to `RequestLike` is primarily just a helper so that users see `Router<RequestLike>` as the type instead of `Router<unknown>` which isn't obvious what the generic does.

Reply:
Solid.